static/bidder-params/bizzclick.json

```diff
@@ -9,11 +9,19 @@
       "description": "Account id",
       "minLength": 1
     },
-    "placementId": {
+    "sourceId": {
       "type": "string",
-      "description": "PlacementId id",
+      "description": "Source id",
+      "minLength": 1
+    },
+    "host": {
+      "type": "string",
+      "description": "Server region",
       "minLength": 1
     }
   },
-  "required": ["accountId", "placementId"]
+  "required": [
+    "accountId",
+    "sourceId"
+  ]
```

Comment:
@BizzClick , `placementId` was required param and PR is renaming `placementId` to `sourceId`.  

This will be an breaking change for the publishers making call to adapter. Should consider a transition period before making such change. Transition period will give provide enough time for the publishers and PBS host compaines to accomadate for such change.

As of now will recommend to keep `placementId`  and mark it as deprecated in bidder docs.  Alsoto `anyone`  as shown below for required params

```
  "oneOf": [
    { "required": ["accountId", "placementId"] },
    { "required": ["accountId", "sourceId"] }
  ]
```

Reply:
Related commit in [doc](https://github.com/prebid/prebid.github.io/pull/5026/commits/85fc3f31c85d388950e592de958b5ebb8f61571c)